python/ray/train/gbdt_trainer.py

```diff
@@ -59,9 +60,38 @@ def _convert_scaling_config_to_ray_params(
             "num_actors": int(num_actors),
         }
     )
-    ray_params = ray_params_cls(
+
+    # This should be upstreamed to xgboost_ray,
+    # but also left here for backwards compatibility.
+    if not hasattr(ray_params_cls, "placement_strategy"):
+
+        class RayParamsFromScalingConfig(ray_params_cls):
+            placement_strategy: Optional[str] = None
+            pgf_args: Optional[Tuple[Any]] = None
+            pgf_kwargs: Optional[Dict[str, Any]] = None
+
+            def get_tune_resources(self) -> PlacementGroupFactory:
+                pgf = super().get_tune_resources()
+                extended_pgf = PlacementGroupFactory(
+                    pgf.bundles,
+                    strategy=self.placement_strategy,
+                    *(self.pgf_args or []),
+                    **(self.pgf_kwargs or {}),
+                )
+                extended_pgf._head_bundle_is_empty = pgf._head_bundle_is_empty
+                return extended_pgf
+
+        ray_params_cls_extended = RayParamsFromScalingConfig
+    else:
+        ray_params_cls_extended = ray_params_cls
+
+    ray_params = ray_params_cls_extended(
         **ray_params_kwargs,
     )
+    scaling_config_pgf = scaling_config.as_placement_group_factory()
+    ray_params.placement_strategy = scaling_config.placement_strategy
+    ray_params.pgf_args = scaling_config_pgf._args
+    ray_params.pgf_kwargs = scaling_config_pgf._kwargs
```

Comment:
IMO it's not great to convert the scaling config into a PGF here and then cherry-pick specific attributes to pass into the PGF.

I understand that this ensures other arguments (e.g. cpu fraction per node) are automatically caught, but maybe for now we can just build the `placement_options` above manually. Feel free to add a comment into `ScalingConfig.as_placement_group` to keep in sync with GBDT RayParams